properties/light.py

```diff
@@ -24,6 +24,11 @@
     "this feature and uses only the light gain"
 )
 
+NORMALIZEBYCOLOR_DESCRIPTION = (
+    "Normalize intensity by the Color Luminance; Radiometric units like Power should not be "
+    "normalized, while Photometric units like Lumen, Candela, Lux should."
```

Comment:
Descriptions should not end with a dot, Blender adds it afterwards (and warns if there is already a dot).